### PR TITLE
DTSPO-3187 - SBOX - Multi-AZ Application Gateway

### PR DESCRIPTION
**Pull request by chrisorisawayi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-3187

### Change description ###
Added new app gateway components (with multi-AZ enabled) for side by side deployment with existing app gateway
Removed non-used 2nd private IP address from sbox appgateway

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: components/appgateway/main.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+module "logworkspace" {
+  source      = "git::https://github.com/hmcts/terraform-module-log-analytics-workspace-id.git?ref=master"
+  environment = var.env
+
```
Comment (nickmoyni) [inline components/appgateway/main.tf:4]: Remove blank line

Comment (chrisorisawayi) [reply components/appgateway/main.tf]: removed - though does not affect tf format or lint checks

**Review by nickmoyni — APPROVED** [review]
LGTM - thanks for fixing the blank line so quickly.

**Review by nickmoyni — APPROVED** [review]
Reviewed changes they look fine.

**Pull request merged**